UI/scripts/view.js

```diff
@@ -0,0 +1,90 @@
+
+const dateConfig = {
+  year: 'numeric',
+  month: 'numeric',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+};
+
+class View {
+  constructor(model, isLoggedUser) {
+    this.photoPosts = model;
+    this.isLoggedUser = isLoggedUser;
+  }
+
+  addPost(post) {
+    if (this.photoPosts.addPhotoPost(post)) {
+      const wrapper = document.querySelector('#wrapper');
+      const newPost = View.getNodeWithHtml(post);
```

Comment:
Naming: better to give names that describes what function do in general then what happened technically.
Here is `buildPost` or smth like that.